search/search-service.test.js

```diff
@@ -24,15 +24,19 @@ describe('search-service', () => {
     });
 
     it('Should call get with correct URL and headers', async () => {
+        const fakeLogger = { info: () => {}, error: () => {}, warn: () => {}, debug: () => {} };
         const service = createSearchService({
             caseReferenceNumber: '12-345678',
-            createRequestService: mockCreateRequestService
+            createRequestService: mockCreateRequestService,
+            logger: fakeLogger // <-- use 'logger' here
         });
         const query = 'example';
         const pageNumber = 2;
         const itemsPerPage = 5;
 
-        const result = await service.getSearchResults(query, pageNumber, itemsPerPage);
+        const req = { headers: { cookie: 'session=abc123' } };
+        const result = await service.getSearchResults(query, pageNumber, itemsPerPage, req);
```

Comment:
Avoid passing the whole request, pass what we need.

Why is the cookie needed for the request inside `getSearchResults`?